src/App.css

```diff
@@ -0,0 +1,350 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400&family=Roboto+Slab:wght@100;200;300;400&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Raleway:wght@100;300;400;500;600&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@200;400;600&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: "Montserrat", sans-serif;
+}
+
+.flex {
+  display: flex;
+  align-items: center;
+}
+
+.p2 {
+  padding: 20px;
+}
+
+.left {
+  text-align: left;
+}
+
+.h1 {
+  font-weight: bold;
+}
+
+.font-2 {
+  font-family: "Roboto Slab", serif;
+}
+
+.font-3 {
+  font-family: "Raleway", sans-serif;
+  font-weight: 300;
+}
+
+body {
+  background: #d4679d;
+  background-image: linear-gradient(to right, #d353ac, rgb(209, 87, 154));
+}
+
+/* .search-field {
+  width: 100%;
+  outline: 0;
+  padding: 5px;
+  border: none;
+  border-radius: 5px;
+} */
```

Comment:
- [ ] I think it would be better to remove unused code from your project rather than comment it out to keep your code clean and follow [the best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)💯

![image](https://user-images.githubusercontent.com/45231276/181750281-0d99a210-25c7-4b54-b455-141811a9c361.png)

Reply:
Thanks for the constructive feedback.
Issues resolved!